pkg/provider/cert_authority_test.go

```diff
@@ -22,29 +22,30 @@ func TestNewCA(t *testing.T) {
 }
 
 func BenchmarkNewCA_Difficulty8_Concurrency1(b *testing.B) {
+	context.Background()
 	for i := 0; i < b.N; i++ {
 		expectedDifficulty := uint16(8)
-		NewCA(nil, expectedDifficulty, 1)
+		NewCA(context.Background(), expectedDifficulty, 1)
```

Comment:
for tests, just fyi, we've been making a global ```var ctx = context.Background()``` so we try and avoid as much as possible having context.Background() be something people get in the habit of calling